packages/react/test/specs/lib/felaRenderKeyframesPlugin-test.ts

```diff
@@ -0,0 +1,59 @@
+import felaRenderKeyframesPlugin from 'src/lib/felaRenderKeyframesPlugin'
+import { felaRenderer } from 'src/lib'
+
+const renderKeyframesPlugin = felaRenderKeyframesPlugin()
+
+describe('felaRenderKeyframesPlugin', () => {
+  test('does not transform the animationName prop if it is already string', () => {
+    const style = {
+      animationName: 'k1',
+      animationDuration: '2s',
+    }
+
+    expect(renderKeyframesPlugin(style, 'RULE', felaRenderer)).toMatchObject(style)
+  })
+
+  test('transforms the animationName prop if it contains keyframe in the definition', () => {
+    const style = {
+      animationName: {
+        keyframe: () => ({ from: { rotate: '0deg' }, to: { rotate: '360deg' } }),
+      },
+      animationDuration: '2s',
+    }
+
+    expect(renderKeyframesPlugin(style, 'RULE', felaRenderer)).toMatchObject({
+      animationName: expect.any(String),
+      animationDuration: '2s',
+    })
+  })
+
+  test('transforms the animationName prop if it contains keyframe in the definition', () => {
+    const style = {
+      animationName: {
+        keyframe: () => ({ from: { rotate: '0deg' }, to: { rotate: '360deg' } }),
+        params: {},
+      },
+      animationDuration: '2s',
+    }
+
+    expect(renderKeyframesPlugin(style, 'RULE', felaRenderer)).toMatchObject({
+      animationName: expect.any(String),
+      animationDuration: '2s',
+    })
+  })
+
+  test('calls the renderer with the keyframe and params object', () => {
+    const params = { from: '0deg', to: '360deg' }
+    const style = {
+      animationName: {
+        keyframe: () => ({ from: { rotate: '0deg' }, to: { rotate: '360deg' } }),
+        params,
+      },
+      animationDuration: '2s',
+    }
+
+    const renderer = { renderKeyframe: jest.fn() }
+    renderKeyframesPlugin(style, 'RULE', renderer)
+    expect(renderer.renderKeyframe).toHaveBeenCalledWith(expect.any(Function), params)
```

Comment:
Not sure how to check here that the keyframe was actually the first param, as the animationName is replaced with string after the plugin is invoked, so I cannot use the same reference. I tried extracting the keyframe function in a const, but then again jest reports it as a different function.